src/fides/cli/commands/util.py

```diff
@@ -189,6 +137,12 @@ def up(ctx: click.Context, no_pull: bool = False) -> None:
         start_application()
         seed_example_data()
         click.clear()
+
+        # Deployment is ready! Perform the same steps as `fides init` to setup CLI
+        echo_green("Deployment successful! Initializing fides...")
+        config_path = create_config_file(ctx=ctx, fides_directory_location=".")
+        print_divider()
+        check_and_update_analytics_config(ctx=ctx, config_path=config_path)
```

Comment:
This now effectively _inlines_ the logic from `fides init`, except without the extra ASCII art and the analytics event. That means a single `deploy` command will setup the sample project, initialize CLI, and prompt for consent/registration 👍